config/mcus/can_ebb36.cfg

```diff
@@ -0,0 +1,28 @@
+[mcu toolhead]
+# This board work by using a serial connection by default.
+# If you want to use CAN, invert the commented lines here
+
+# serial: /dev/serial/by-id/change-me-to-the-correct-mcu-path
+# canbus_uuid: change-me-to-the-correct-canbus-id
```

Comment:
Can you replace the header ? This board is a dedicated CAN board and it will be more clean and similar to what we have in the other file `canboard_SHT36-42.cfg`.

```
##--------------------------------------------------------------------
canbus_uuid:  change-me-to-the-correct-can-id
##--------------------------------------------------------------------
```

BTW, can you also change the name of the file to `canboard_EBB36-42.cfg` ?